### ISSUE TITLE
Can't package Boost server while it is started

### ISSUE THREAD

**Issue by scottkurz** [OpenLiberty/boost#132]
Tried to back up to show symptom.. am stuck in a Git issue.  Will do later.

I think the fix might be as simple as adding:
```java
         executeMojo(getPlugin(), goal("package-server"),
                 configuration(element(name("isInstall"), "false"), element(name("include"), "minify,runnable"),
                         element(name("attach"), Boolean.toString(attach)),
+                        element(name("outputDirectory"), "target/liberty-alt-output-dir"),
```

like we do in LMP [parent POM](https://github.com/WASdev/ci.maven/blob/61c3044d6b1598c33325b7d59c2964756276badb/liberty-maven-app-parent/pom.xml#L87-L96)

**Comment by chyt** [comment]
I thought that this was a limitation within Liberty? My understanding is that a server cannot be packaged while it is running. If you try, you get this error message from `bin/server package <servername>`:

> Server <servername> package failed. It must be stopped before it can be packaged.

**Comment by chyt** [comment]
Closed by #155. However there is still some ongoing discussion in that thread.
